runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean chatboxTooltip()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 2,
```

Comment:
Fix your formatting (check codestyle on our wiki)

Reply:
Reformatted in commit 3dea92c9422f4966c97188bf53c48d803b75e0c8